rtvi-client-js/src/client.ts

```diff
@@ -165,21 +165,19 @@ export type RTVIEventCallbacks = Partial<{
   onTrackStopped: (track: MediaStreamTrack, participant?: Participant) => void;
   onLocalAudioLevel: (level: number) => void;
   onRemoteAudioLevel: (level: number, participant: Participant) => void;
+
   onBotStartedSpeaking: (participant: Participant) => void;
   onBotStoppedSpeaking: (participant: Participant) => void;
   onUserStartedSpeaking: () => void;
   onUserStoppedSpeaking: () => void;
-
   onUserTranscript: (data: TranscriptData) => void;
-  onBotTranscript: (data: TranscriptData) => void;
-  onUserText: (text: UserLLMTextData) => void;
-  onBotText: (text: BotLLMTextData) => void;
-  onBotLlmStarted: (participant: Participant) => void;
-  onBotLlmStopped: (participant: Participant) => void;
-
-  onBotTtsText: (text: TTSTextData) => void;
-  onBotTtsStarted: (participant: Participant) => void;
-  onBotTtsStopped: (participant: Participant) => void;
+  onBotTranscript: (data: BotLLMTextData) => void;
+  onBotLlmText: (data: BotLLMTextData) => void;
+  onBotLlmStarted: () => void;
+  onBotLlmStopped: () => void;
```

Comment:
What is the `participant` in `onBotStartedSPeaking`? Should that be removed as well?

Reply:
@aconchillo you might be right, but I thought we did receive a participant object there. Let me check and update accordingly.

Reply:
Confirmed these are not implemented. Removed.